lib/src/room.dart

```diff
@@ -612,14 +615,42 @@ class KurentoRoom extends ChangeNotifier implements Room {
           await _updateServiceRequestStatus('ASSIGNED', agentFirstName);
         } else if (_serviceRequestState == ServiceRequestState.assigned) {
           _serviceRequestState = ServiceRequestState.started;
-          notifyListeners();
 
           // Once we've started, we no longer need the timer to consume resources. This means we may miss an
           // Agent-initiated end message, but the impact of that is low (the Explorer can end the call themselves).
           _getServiceRequestStatusTimer?.cancel();
 
+          // HACK: This condition is here to cover the improbable possibility
```

Comment:
I don't love this hack and would be okay shipping without it. 

If we do keep it, I think we should add a task to remove it after collecting some data.